### DISCUSSION TITLE
Options

### DISCUSSION THREAD

**Discussion by alboxe** [pxlrbt/filament-excel#176] [Q&A] [answered]
I want to change the export type to csv. My Filament code looks like this:
          ....
            ->bulkActions([
                Tables\Actions\BulkActionGroup::make([
                    ExportBulkAction::make()->exports(
                        [ExcelExport::make()->withWriterType(\Maatwebsite\Excel\Excel::CSV) ]
                    ),
                    Tables\Actions\DeleteBulkAction::make(),
                ]),
            ])
I get a csv file, but it's empty. Do I have to put the code for withWriterType() somewhere else? The default
ExportBulkAction::make() is doing fine.
Please a reaction
Kr.
Albert Oxener

**Comment by pxlrbt** [comment, marked as answer]
No, the code is fine. But you defined your own Export and you didn't specify any columns that you want to export. See `->withColumns()`, `->fromTable()`, etc.

**Comment by alboxe** [comment]
Hi Dennis,
Thanks for the quick reaction. Defining the columns was the solution.
Albert

Op vr 9 feb 2024 om 11:19 schreef Dennis Koch ***@***.***>:
